unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FeatureController/FeatureController.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using DCL;
+using UnityEngine;
+
+public class FeatureController
```

Comment:
Is this class something for only the BuilderInWorld feature or for other ones too? Can we add some kind of generic documentation in the class header describing its main responsibility?

Reply:
This is something more generic. Sure! I will add some documentation to both parts